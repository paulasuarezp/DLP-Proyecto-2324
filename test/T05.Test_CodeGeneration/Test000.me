class Test000; /* Hola Mundo!
/*/-------------------------------

	create main;
	
/*/-------------------------------

	feature inicio is
		do
            print 'H', 'o', 'l', 'a', ' ', 'M', 'u', 'n', 'd', 'o';
            println '!';
	end

	feature main is 
		do
			inicio();
	end

end

run	main();