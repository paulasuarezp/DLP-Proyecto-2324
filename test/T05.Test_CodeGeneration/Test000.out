#source "Test000.me"

'Declaraciones globales

call main
halt

'**
'** Features
'**

#line 8
inicio:
#func inicio
#ret void

#line 10
pushb 72
outb
pushb 111
outb
pushb 108
outb
pushb 97
outb
pushb 32
outb
pushb 77
outb
pushb 117
outb
pushb 110
outb
pushb 100
outb
pushb 111
outb

#line 11
pushb 33
outb
pushb 10
outb

#line 12
ret 0, 0, 0

'**
'** Main
'**

#line 17
main:
#func main
#ret void

#line 17
call inicio

#line 18
ret 0, 0, 0
